trunner/test_runner.py

```diff
@@ -50,6 +51,8 @@ def run(self):
         self.parse_tests()
 
         for test_config in self.test_configs:
+            if test_config["type"] == 'busybox' and not self.long_test:
```

Comment:
We are mixing logic here. If there is long-test flag and type of the test is busybox then we should dismiss it at the parsing step.

Reply:
I don't quite understand. I don't want to run busybox tests when the `long-test` argument is not specified, so I think the condition is ok. Please correct me if am I wrong. But that's right it can be checked earlier, I will correct it.